Makefile

```diff
@@ -55,6 +55,7 @@ mypy:
 	# Again, we are starting small, detecting only 'BlockNumber' related errors,
```

Comment:
This comment needs a small update as well.

Reply:
I missed it.  Updated.